wiki/Mapping_Techniques/Basics/en.md

```diff
@@ -1,25 +1,150 @@
-Flow of beatmap design
-----------------------
+# Basics
```

Comment:
The title will look like this <https://new.ppy.sh/wiki/People/Beatmap_Nomination_Group/>
(note that `People` is still here while `Beatmap Nomination Group` is the title)

If I name it to `Basics of Mapping Techniques` then it will show
`Mapping Techniques` then `Basics of Mapping Techniques`